### PR TITLE
Added Flick HAT and Zero

### PR DESCRIPTION
**Pull request by francesco-vannini**
Modified JustBoom Amp Zero
Renamed and modified details for Pi PoE this affects the png name too

### CODE REVIEW

**Comment by francesco-vannini** [comment]
Hopefully this is now OK. I had indicated changes to the JustBoom Amp Zero and the PiPoE in the previous PR and thought that those needn't be in the draft folder so that it would show what the changes were when merging. Let me me know if there is anything further that needs to be done.

File: draft/overlay/flick-zero.md [hunk 1]
```diff
@@ -0,0 +1,51 @@
+<!--
+---
+name: My Add-on Board
```
Comment (Gadgetoid) [inline draft/overlay/flick-zero.md]: Needs to change to "Flick Zero"

File: draft/overlay/flick-hat.md [hunk 2]
```diff
@@ -0,0 +1,51 @@
+<!--
+---
+name: My Add-on Board
```
Comment (Gadgetoid) [inline draft/overlay/flick-hat.md]: Needs to change to "Flick HAT"

**Comment by francesco-vannini** [comment]
Hey guys, is there anything else I can help with on this? Can we move this forward?

**Comment by francesco-vannini** [comment]
Hi @Gadgetoid thanks for moving this forward, really appreciated. There were however also changes to the JustBoom Amp Zero and the Pi PoE which don't seem to appear on the production website. Could you add those when you have a minute?

Essentially for the JustBoom Zero this was to be removed:
```
-'15':
-name: Soft Mute
```
and for the Pi PoE (including the picture rename):
```
-formfactor: Custom
+formfactor: HAT

-image: 'pisupply-pi-poe.png'
+image: 'pi-poe.png'

-eeprom: setup
+eeprom: yes
```

**Comment by francesco-vannini** [comment]
Hi @Gadgetoid would it be OK then to add those remaining changes?

**Comment by shawaj** [comment]
They are in the commits - https://github.com/Gadgetoid/Pinout.xyz/pull/229/files

But do these commits just go to a draft and then they have to be pulled into the live site?

**Comment by Gadgetoid** [comment]
We don't really have a good way to deal with edits on existing boards- since once published they are copied to multiple languages. I think the best practise in future would be to make the changes to the published board files for all languages (painful, but someone has to do it at some point) and submit that as a PR.

I'll use a diff tool to resolve those changes and get them published as soon as I can.

**Comment by shawaj** [comment]
@gadgetoid so we just look in here - https://github.com/Gadgetoid/Pinout.xyz/tree/master/src
For any translated versions and update those too, if they exist already?

What if they were more "involved" changes which required more actual translation?

**Pull request merged**